tests/integration_test.go

```diff
@@ -664,7 +664,7 @@ func (s *IntegrationTestSuite) testAbortXMRMakerCancels(asset types.EthAsset) {
 
 	peerIDs, err := c.Discover(string(coins.ProvidesXMR), defaultDiscoverTimeout)
 	require.NoError(s.T(), err)
-	require.Equal(s.T(), 1, len(peerIDs))
+	require.Equalf(s.T(), 1, len(peerIDs), "peer count mismatch")
```

Comment:
It was easy to look at the test output and incorrectly assume that the offer count at the end did not match up, even though the test was actually failing here. I added messages to clarify.

As to why `TestAbort_ERC20_XMRMakerCancels` would fail, but not `TestAbort_XMRMakerCancels`, I did some research on testify's `Suite` and it runs the tests in alphabetical order. It is not because `Suite` intentionally alphabetizes the tests, but rather because the standard library reflect methods used to discover the functions return them in alphabetical order.

`TestAbort_ERC20_XMRMakerCancels` is always the very first test to run (of *all* the tests) and while the `swapd` daemons were fully started, peer discovery had not completed. I added more time in `run-integration-tests.sh` after starting Charlie and Bob's daemons.